src/hooks/use-sync-states-progress-info.ts

```diff
@@ -105,6 +107,16 @@ export function useSyncStatesProgressInfo() {
 				progress: 40,
 				message: __( 'Uploading Studio site…' ),
 			},
+			uploadingPaused: {
+				key: 'uploadingPaused',
+				progress: 45,
+				message: __( 'Uploading paused, retrying…' ),
```

Comment:
```suggestion
				message: __( 'Uploading paused' ),
```
Given that we have the tooltip, I think `Uploading paused` is enough.

Reply:
Makes sense. Fixed in [4b3c2bf](https://github.com/Automattic/studio/pull/2212/commits/4b3c2bf97fe1cbfa37fd2aee26d83ea1c5748323).